### PR TITLE
Add peek_first_item() and peek_last_item()

### PR DESCRIPTION
**Pull request by normanyahq**
I've implemented these two methods for my personal use, it might be useful.

### CODE REVIEW

File: lru.c [hunk 1]
```diff
@@ -333,6 +333,34 @@ get_key(Node *node)
 }
 
 static PyObject *
+LRU_peek_first_item(LRU *self)
+{
+    PyObject *tuple = PyTuple_New(2);
```
Comment (amitdev) [inline lru.c]: Create the tuple inside the if block, otherwise it might leak memory.

File: lru.c [hunk 2]
```diff
@@ -333,6 +333,34 @@ get_key(Node *node)
...
+        PyTuple_SET_ITEM(tuple, 1, self->first->value);
+        return tuple;
+    }
+    else Py_RETURN_NONE;
+}
+
+static PyObject *
+LRU_peek_last_item(LRU *self)
+{
+    PyObject *tuple = PyTuple_New(2);
```
Comment (amitdev) [inline lru.c]: Same as above

**Comment by amitdev** [comment]
Thanks @normanyahq. Please check my comment and then I'll merge it.

**Comment by normanyahq** [comment]
Fixed. Thank you for your suggestion.

**Pull request merged**